cmd/knoxite/utils/utils.go

```diff
@@ -177,7 +177,7 @@ func PathToUrl(u string) (*url.URL, error) {
 	// In case some other path elements have wrongfully been interpreted as Host
 	// part of the url
 	if url.Host != "" {
-		url.Path = url.Host[1:] + url.Path
+		url.Path = url.Host[:] + url.Path
```

Comment:
Didn't we do this to strip the initial, spurious slash from `file:///`? I fear this breaks Windows builds again, so we need to revisit  that. Regardless, in this case it should probably just read `url.Host + url.Path`.